cores/esp32/HardwareSerial.h

```diff
@@ -93,16 +93,15 @@ class HardwareSerial: public Stream
     void setDebugOutput(bool);
 
 protected:
+    void (*intr)() ;
     int _uart_nr;
     uart_t* _uart;
 };
 
-extern void serialEventRun(void) __attribute__((weak));
-
```

Comment:
This needs to be restored as this weakly declared method is called, this also breaks the tests which prevent this PR from being considered for merge.

Reply:
This was taken care of in #2991 prior to this :-)